scripts/aem.js

```diff
@@ -409,6 +411,63 @@ function wrapTextNodes(block) {
   });
 }
 
+/**
+ * Opens an iframe with the video when clicking on anchor tags.
+ * @param {Element} element container element
+ */
+
+function openVideoIframe(element) {
```

Comment:
aem.js is a "framework" file, in general we should avoid modifying as much as possible. I'd suggest moving this logic to scripts.js for now.